UNITTESTS/unit_test/test.py

```diff
@@ -86,6 +86,7 @@ def create_makefiles(self,
         args = [cmake,
                 "-G",
                 generator,
+                "-DBUILD_TESTING=ON"
```

Comment:
I would like to avoid major rework on this script as we are proposing to use CMake/CTest command for unittest from Mbed CLI 2 and deprecate this python module scripts along with Mbed CLI 1 and time being these changes to make backward compatibility with Mbed CLI 1